pyproject.toml

```diff
@@ -18,7 +18,6 @@ classifiers = [
     "Intended Audience :: Developers",
     "Operating System :: Unix",
     "Topic :: Software Development :: Quality Assurance",
-    "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
```

Comment:
add 3.13 here if it is compatible/supported?

Reply:
It was already there (2 lines below).

Reply:
ah my bad, cheers!